docs/_layouts/global.html

```diff
@@ -30,6 +30,8 @@
         <script src="js/vendor/modernizr-2.6.1-respond-1.1.0.min.js"></script>
 
         <link rel="stylesheet" href="css/pygments-default.css">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
```

Comment:
For all the external css/js files for Spark UI/Doc, we always download it and make it as part of the source code.
I think this is to make sure Spark UI/Doc can be run and access in a private network.
However, it seems fine for Spark Doc. cc @srowen @sarutak , WDYT?

Reply:
In my experience, HistoryServer is used within a closed network so I think it's better to make css/js as part of source code.
On the other hand, I think it's fine for docs to use external link.

Reply:
Yeah. the `docsearch.js` maybe associate with external search engines. It's not good to copy as part of source code.